fix broken links, remove wikilinks (#4712)
audacity/audacity: CONTRIBUTING.md

## Developing

> Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable in it to start contributing code.

Reviewer: [Proofreading]
```suggestion
Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable with using this programming language to start contributing code.
```

> If you want to develop Audacity:

Reviewer: [Nitpick] I think adding the verb “help” would facilitate a more collaborative working environment for this project.
```suggestion
If you want to help develop Audacity:
```

> * Build instructions can be found in the BUILDING.md

Reviewer: [Typo]
```suggestion
* Building instructions can be found in the BUILDING.md
```

You can download current development builds in the [Actions tab on Github](https://github.com/audacity/audacity/actions). These builds are tied to certain pull requests. As a rule of thumb: If it's not yet merged, report any broken things you find which might be a result of the pull request directly as a comment on the pull request. If the code has been merged, [make a new bug in the issue tracker](https://github.com/audacity/audacity/issues/new/choose).

When reporting bugs, try to find the most general form of it. For example, if you encounter a bug when amplifying a clip 2 hours into a project, try to see if it also happens when you're using a different effect (Normalize, for example), and if it also happens if the clip is near the beginning.

> Bugs must be reproducible. If you can't find steps to reproduce a bug, try asking if you went wrong somewhere [in the Audacity forum](https://forum.audacityteam.org/) instead. 

Reviewer: [Proofreading]
```suggestion
Bugs **must** be reproducible. If you can't find steps to reproduce one, try posting about your experience [in the Audacity forum](https://forum.audacityteam.org/) instead. 
```

## Translating

See [our translators page](https://www.audacityteam.org/community/translators/) for more information on how to translate Audacity. 

## Feedback & Feature requests

> If you have anything you thing Audacity can do better, you can voice it in [the relevant section in the forum](https://forum.audacityteam.org/c/feedback-and-discussion-forum/adding-features-to-audacity/22). If you have a very concrete idea something in the code which should be added or changed, you can also  [make an enhancement request in the issue tracker](https://github.com/audacity/audacity/issues/new/choose).

Reviewer: [Proofreading]
```suggestion
Got some ideas on how to improve Audacity? Share them at [the relevant section in the forum](https://forum.audacityteam.org/c/feedback-and-discussion-forum/adding-features-to-audacity/22). If you have a very concrete idea like something in the code which should be added or changed, you can also  [make an enhancement request in the issue tracker](https://github.com/audacity/audacity/issues/new/choose).
```
